src/services/Carts.php

```diff
@@ -537,4 +538,27 @@ private function _getCartPaymentCurrencyIso(): string
 
         return Plugin::getInstance()->getPaymentCurrencies()->getPrimaryPaymentCurrencyIso();
     }
+
+    /**
+     * @param User $user
+     * @return void
+     * @throws MissingComponentException
+     * @throws Throwable
+     */
+    public function customerRegisteredHandler(User $user): void
```

Comment:
Should we name this similar to the other handlers? Just in case we want to use it for more than one purpose in the future?

`afterSaveUserHandler` would be my suggestion